Dockerfile

```diff
@@ -24,6 +24,9 @@ RUN yum update -y \
 RUN pip install -U pip wheel
 
 COPY requirements /pip/requirements/
+
+RUN sed -i 's/simplejson==2.3.2/simplejson>=2.3.2/' /pip/requirements/compiled.txt
```

Comment:
I'll test using the version that is being removed (3.0.5) and see if that builds without error.